tests/atomistics/job/test_sqs.py

```diff
@@ -19,9 +19,14 @@ def tearDownClass(cls):
         cls.project.remove(enable=True, enforce=True)
 
     def test_run(self):
-        self.project.create_job(
-            'SQSJob', "job_test"
-        )
+        try:  # Only test if the machine has access to sqsgenerator -- at time of writing Windows doesn't
```

Comment:
I do not think this is necessary unless you plan to actually run the calculation. But for running the calculation we would have to also install the SQSgenerator which is currently not done.

Reply:
Since sqsgenerator only runs on linux, I'd suggest wrapping in
```python
import sys
if sys.platform == 'linux':
  ...
```
That makes the intent clearest.

Reply:
No, it runs on Linux and Mac OS X

Reply:
> I do not think this is necessary unless you plan to actually run the calculation. 

I was actually not super happy that it didn't fail until runtime, [so I changed it to fail earlier](https://github.com/pyiron/pyiron/pull/1023/commits/5997fb705e9e8f005b4f312d23ae0fdc88bf1f2e). Thus the test needs it when the job is created.

> But for running the calculation we would have to also install the SQSgenerator which is currently not done.

You mean we don't have `sqsgenerator` on our Linux and Mac test environments? I mean, I guess I'll find out in <50m when the test passes or fails... I would like to have them though, since we expect `SQSJob` to be running on Linux and Mac environments and are testing it...

> That makes the intent clearest.

Howso? I think we don't care what system we're on, but that whatever the system is it can import the module. In this sense I find Jan's original try-except clear. (Ok, we pass the failure, which is a little unclear...I will add a comment clarifying intent!)